frontend/packages/dev-console/integration-tests/features/gitops/install-GitOps-Operator-ODC-3067.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Install GitOps Service Operator
+    User should be able to install the GitOps Service Operator
+
+
+Background: 
+    Given user is at the Administrator perspective
+
+
+@regression
+Scenario: Navigation bar when GitOps Operator is not installed
+    Given user is at Developer perspective
+    Then user will not see GitOps nav Item
+
+
+@regression
+Scenario: Install GitOps Operator
+    Given user is at OperatorHub
+    When user searches for GitOps Service Operator
+    And user clicks on the GitOps Service card
+    And user clicks on the install button
+    And user clicks on install GitOps Operator button
+    Then user will see that GitOps Service Operator is installed
```

Comment:
Can we mention over here about the page where it will be seen installed i.e. GitOps Service operator is seen installed in installed operator page?

Reply:
Updated to new statement